cnf-tests/testimages/images.json

```diff
@@ -1,22 +1,10 @@
 [
```

Comment:
There's no `testimages` anymore. Would it make sense to rename the dir to "testbinaries" and move this file one level up?